browser-tests/utils/jwt/jwt.ts

```diff
@@ -34,7 +34,7 @@ export function getEpochTimeframeForTestJWt(
 export function generateTestJwt({
   user,
   prefix,
-  issuer = 'https://kukkuu-admin.test.hel.ninja',
```

Comment:
The default value could be get from the config https://github.com/City-of-Helsinki/kukkuu-admin/blob/master/browser-tests%2Futils%2Fjwt%2Fconfig.ts#L37

Reply:
This will be fixed on the another ticket